### DISCUSSION TITLE
cz fetches "rc" tag instead of "alpha" tag

### DISCUSSION THREAD

**Discussion by ramnarayan-code** [commitizen-tools/commitizen#985] [Q&A]
Hi cz community, 

First of all, thanks a lot to the community for the release of v3.14.0 which fixed the version bump between the pre-releases. 

But we have another issue. 

In our setup, we create "alpha" tags for "dev" env, "rc" tags for "stage" env and final release for "prod" env. 

When we follow the below workflow,
Step 1-> Start with 1.0.0 -> New "feat" commits in feature branch –> 1.1.0a0(Dev) -> 1.1.0rc0(Stage)​
Step 2-> New "feat" commits in feature branch -> 1.1.0rc1(Dev) (_Wrong bump: 1.1.0rc0->1.1.0rc1, Expected bump: 1.1.0a0->1.1.0a1_)​

Despite passing "--prerelease alpha" to cz bump in dev branch, it fetched 1.1.0rc0 tag and bumped to 1.1.0rc1 during the ci_cd pipeline run for dev branch.

The expectation is when prerelease argument is passed, cz is supposed to fetch the corresponding latest tag.

Looking forward to your suggestions.

**Comment by ramnarayan-code** [comment]
@woile It will be great and helpful if you could clarify this question. Looking forward to your response

**Reply by woile** [reply]
Are you using the SCM provider? What CI are you using? can you show the pipeline for dev. I think you should make sure that on the pipeline for dev you should only pull the tags for dev.

**Reply by ramnarayan-code** [reply]
**Are you using the SCM provider?**
Yes having the config in pyproject.toml
`version_provider="scm"
`
 **What CI are you using?**
Azure DevOps

The command I use for dev branch is 
`cz bump --yes --git-output-to-stderr --annotated-tag --prerelease alpha --tag-format "$major.$minor.$patch$prerelease"`

I do not have any specific settings in Azure DevOps ci_cd config to pull only alpha tags for dev.

**Reply by ramnarayan-code** [reply]
@woile 
Could you please clarify when the "--prerelease alpha" argument is set, cz is supposed to fetch only alpha tags and bump the version with alpha suffix only?
